pyperformance/tests/data/find-pyperformance.py

```diff
@@ -13,7 +13,7 @@
 testsroot = os.path.dirname(datadir)
 pkgroot = os.path.dirname(testsroot)
 reporoot = os.path.realpath(os.path.dirname(pkgroot))
-marker = os.path.join(reporoot, 'setup.py')
+marker = os.path.join(reporoot, 'pyproject.toml')
```

Comment:
This seems unrelated?

Reply:
I wanted to run tests and this was a blocker for running them. I left this change in a different commit so if the PR is merged without squashing this change and the next one will be in a separate commit.
If you want I can create a separate PR to include these to small changes.

Reply:
This is the commit: https://github.com/python/pyperformance/pull/351/commits/f47ab7c5d0392cb8c71b53a9dc557076e1ab7bf7

Reply:
Dropped, I'll raise a new PR